src/domains/tasks/ViewTasks/components/tests/OffersSection.test.tsx

```diff
@@ -0,0 +1,34 @@
+import { screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+
+import { OffersSection } from "../TasksPage";
+
+import { renderWithPolarisTestProvider } from "src/test/utils";
+
+function renderOffersSection() {
+  return renderWithPolarisTestProvider(<OffersSection />);
+}
+
+test("Smoke test if it renders", () => {
+  const { baseElement } = renderOffersSection();
+
+  expect(baseElement).toBeInTheDocument();
```

Comment:
Generally I go with a div with `aria-label` + `ByLabelText` for my smoke tests.

I've opted to go with toBeInTheDocument() because the `OffersSection` is a part of a `TaskCard`.